pkg/admissioncontroller/webhooks/auth/seed/authorizer_test.go

```diff
@@ -196,5 +195,85 @@ var _ = Describe("Seed", func() {
 				Expect(reason).To(BeEmpty())
 			})
 		})
+
+		Context("when requested for ConfigMaps", func() {
```

Comment:
Side-note (not strictly related to this PR): it would be nice to write an integration test for this and try out something with gardener resources in envtest as well.
WDYT? Should we give it a try?

Reply:
Yes, I would love to do so. I talked about it with @timuthy already. We would need to have a similar `envtest` package that also sets up `gardener-apiserver` and then we would be able to do so. Do you have something like this in mind?

Reply:
Yes something like this. Basically compile `gardener-apiserver` in `setup-envtest.sh` and then wrap `envtest` to connect it to the temp etcd.
Alternatively we could generate CRDs from our API types and use them for imitating g-api.
Anyhow, this should be doable, I guess.